src/app/Processor/ProcessorEdit/ConfigurationForm/ConfigurationForm.tsx

```diff
@@ -62,7 +62,13 @@ const ConfigurationForm = (props: ConfigurationFormProps): JSX.Element => {
       disabled={readOnly}
     >
       {Object.keys(properties as { [key: string]: unknown }).map((key) => (
-        <AutoField key={key} name={key} disabled={readOnly} />
+        <AutoField
+          key={key}
+          name={key}
+          disabled={readOnly}
+          data-ouia-component-id={key}
+          data-ouia-component-type="SE/parameter"
```

Comment:
I am little bit unsure about the chosen type name. Please correct me if I am wrong. I think `SE` stands for `Smart Events`. 

As the changes are done in `ProcessorEdit` folder, I am not sure if the component is common for whole `Smart Events` app, or just for `Processors`.

If  we keep your proposal, my expectations would be `Bridge parameters` and `Processor parameters` are visually totally the same and alsi corresponding html has the same naming conventions, is that the case?

Reply:
I am trying to use the namespace definition similar as "PF4" which is the name of the whole project not a part of them. I understand that `parameter` might be a poor naming. @jomarko what do you thing about `SE/config-parameter` or `SE/processor-parameter` I am not sure if I am able to know if it is `action` or `source` parameter at this level of code.